hack/rbac-check/clusterroles.go

```diff
@@ -172,13 +172,13 @@ func withRBACCheckTag(node *yaml.Node) bool {
 // extractRBACCheckTags parses '+rbac-check' tags from a YAML node's comments,
 // and returns them in the format:
 //
-//   map[string]string{
-//     "subresource":"status",
+//   map[string][]string{
+//     "subresource": {"status"},
 //   }
 //
 // The returned map is empty if the parsed tag doesn't contain any key/value,
 // nil if the provided YAML node isn't tagged.
-func extractRBACCheckTags(node *yaml.Node) map[string]string {
+func extractRBACCheckTags(node *yaml.Node) generators.CommentTag {
```

Comment:
After updating `knative/pkg`, the returned `map` has `[]string` as value instead of `string` (to support keys with multiple values), so I had to adjust this function accordingly.